cli/src/build/onboarding/android/ui/app.tsx

```
@@ -64,6 +65,8 @@ import {
   PLAY_DEVELOPERS_URL,
 } from '../play-api.js'
 import { deleteAndroidProgress, getAndroidResumeStep, loadAndroidProgress, saveAndroidProgress } from '../progress.js'
+import { mapAndroidOnboardingError } from '../../error-categories.js'
```

ESLint

[error] Expected "../../error-categories.js" to come before "../progress.js".

(perfectionist/sort-imports)